Samples/Forms/LibVLCSharp.Forms.Sample/MainViewModel.cs

```diff
@@ -1,12 +1,20 @@
 ﻿using LibVLCSharp.Shared;
+using System;
 using System.ComponentModel;
+using System.Threading.Tasks;
 
 namespace LibVLCSharp.Forms.Sample
 {
     public class MainViewModel : INotifyPropertyChanged
     {
         public event PropertyChangedEventHandler PropertyChanged;
-        LibVLC _libVLC;
+
+        public MainViewModel()
+        {
+            Task.Run((Action)Initialize);
```

Comment:
Seems like a hack and can't handle exceptions properly. What's the issue with calling Initialize externally?

Reply:
No, it's not a hack, it's to show that we can handle a true MVVM scenario. All the code (initialization, play, ...) can be done in the ViewModel. I kept some code in the MainPage.xaml.cs file but it' only to handle the WPF case (where Play method must be called after that the HWND is set).
There is no problem to call the Initialize method externally but it breaks the MVVM scenario.
For the exceptions, you can handle them in the Initialize method, what's wrong ?